### ISSUE TITLE
Next.js 9.0.6 breaks the builder with dynamic routes

### ISSUE THREAD

**Issue by tusbar** [vercel/vercel#3040]
I haven’t investigated more than this, but building on Now with dynamic routes throws

```traceback
Error: Found usage of dynamic routes but not on a new enough version of Next.js.
  at Object.getDynamicRoutes (/tmp/396dbae88ede4a65/.build-utils/.builder/node_modules/@now/next/dist/index.js:6034:15)
  at Object.module.exports.178.exports.build (/tmp/396dbae88ede4a65/.build-utils/.builder/node_modules/@now/next/dist/index.js:4247:35)
  at <anonymous>
```
https://github.com/zeit/now/blob/efe114fa8611acdd933d4721a93d40eab1ed4b88/packages/now-next/src/utils.ts#L342-L346

**Comment by tusbar** [comment]
Doesn’t seem to be happening anymore…

**Comment by styfle** [comment]
Thanks! I believe this was fixed in `@now/next` version 1.0.1
